readthedocs.yml

```diff
@@ -0,0 +1,5 @@
+build:
+  image: latest
+python:
+  version: 3.6
+  setup_py_install: true
```

Comment:
Does pip_install let us get away without the requirements.txt?

Reply:
There's also `extra_requirements` if one of the optional requirements was the problem.

Reply:
Ok. Whatever. Reading through their docs and forums it really seems like we shouldn't need anything other than the setup.py but it's certainly not worth spending more time on right now.